MatrixSDK/Data/Store/MXFileStore/MXFileStore.m

```diff
@@ -293,6 +293,27 @@ - (void)diskUsageWithBlock:(void (^)(NSUInteger))block
     });
 }
 
+- (void)logFiles
+{
+    NSLog(@"[MXFileStore] logFiles: Files in %@:", self->storePath);
+    NSArray *contents = [[NSFileManager defaultManager] subpathsOfDirectoryAtPath:self->storePath error:nil];
+    NSEnumerator *contentsEnumurator = [contents objectEnumerator];
+    
+    NSUInteger fileCount = 0, diskUsage = 0;
```

Comment:
Shouldn’t we use `long long` for `diskUsage` and other file size variables?

Reply:
I just copied and pasted the code of the method above. It works enough.